### DISCUSSION TITLE
Questions about the 0.5.0 Update

### DISCUSSION THREAD

**Discussion by ExtraCode75** [FunkinCrew/Funkin#3017] [Q&A] [answered]
I'm just wondering about the milestone bar on some pull requests, and I have been noticing that the bar is around 30% the way through for a month now. Any reason as to why? 

Also side question, How huge will the update be? Pit Stop or just a destination update? (Like new week, or overhaul of new stuff, etc.)

**Comment by JVNpixels** [comment]
You are right! I saw on my story mode menu scrolling pull request EliteMasterEric added the “0.5.0” milestone label, and it said 31% finished and it’s been a month since the PitStop update came out (the latest one). So this may take a couple months to finish!

**Comment by AppleHair** [comment]


**Reply by JVNpixels** [reply]
> I had two of my PRs added to the 0.5.0 milestone and merged to a branch called `feature/playable-pico-mode`, so they might be adding pico's variations to some of the songs for 0.5.0, which I think will take some time.

Where did you find this branch? That seems like it would take alot of time as well.

**Comment by Darkness8374939** [comment]
Seems they will be fixing the duplicate Dadbattle and breakfast songs in the files.

**Comment by JVNpixels** [comment]
How are you guys finding these branches/updates and where?

**Reply by Darkness8374939** [reply]
https://github.com/FunkinCrew/Funkin/milestone/5 (Here)

**Comment by JVNpixels** [comment]
After some digging, I found some code about the pico stuff!

**Comment by NotHyper-474** [comment, marked as answer]
If I'm not mistaken the percentage is based on the total of open issues/PRs and the total of closed/merged issues/PRs on the milestone

```math
\frac{11}{24 + 11} \times 100 \approx 30
```

So it's not really an arbitrary percentage of how much the update is complete

**Comment by AbnormalPoof** [comment]
0.5.0 will be a **Destination / Major** update.
https://www.kickstarter.com/projects/funkin/friday-night-funkin-the-full-ass-game/posts/4153415

**Comment by biomseed** [comment]
The devs don't notice that it's visible or not yet as far as I know, so take the information with a grain of salt.
